package.json

```diff
@@ -7,5 +7,9 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
```

Comment:
Add a build command here
![image](https://user-images.githubusercontent.com/73254040/98376042-91a0e500-2008-11eb-822a-14d1f4834dc0.png)
[Source](https://microverse.pathwright.com/auth/sign-in/?next=/library/fast-track-curriculum/69047/path/step/59622983/)